obspy/signal/spectral_estimation.py

```diff
@@ -2020,7 +2053,48 @@ def __plot_coverage(self, ax):
+               [8.3385376e-06, 4.8130301e-05, 3.1461832e-04, 1.1054180e-03, 3.2015584e-03, 5.9504166e-03, 3.4106672e-03, 1.0389085e-03]],
+    (5.5,10):[[4.4187500e-02, 8.8375000e-02, 1.7670000e-01, 3.5350000e-01, 7.0700000e-01, 1.4140000e+00, 2.8280000e+00, 5.6600000e+00, 1.1300000e+01, 2.2600000e+01],
+              [5.1597812e-05, 2.4350754e-04, 1.5861857e-03, 9.1333683e-03, 5.4321168e-02, 3.0210031e-01, 4.4505525e-01, 3.3287588e-01, 3.0775399e-01, 1.2501407e-01]],
+    (5.5,100):[[4.4187500e-02, 8.8375000e-02, 1.7670000e-01, 3.5350000e-01, 7.0700000e-01, 1.4140000e+00, 2.8280000e+00, 5.6600000e+00, 1.1300000e+01, 2.2600000e+01],
+               [7.6042044e-06, 5.0568931e-05, 2.6926792e-04, 9.7744507e-04, 3.4530700e-03, 9.2832164e-03, 1.5054122e-02, 1.3421025e-02, 8.0524871e-03, 2.0691071e-03]],
+    (6.5,10):[[8.8375000e-02, 1.7670000e-01, 3.5350000e-01, 7.0700000e-01, 1.4140000e+00, 2.8280000e+00, 5.6600000e+00, 1.1300000e+01, 2.2600000e+01],
+              [1.7468292e-02, 6.5551361e-02, 1.8302926e-01, 3.9065640e-01, 7.1420714e-01, 1.0447672e+00, 1.2770160e+00, 1.0372500e+00, 5.0465056e-01]],
+    (6.5,100):[[2.2097000e-02, 4.4187500e-02, 8.8375000e-02, 1.7670000e-01, 3.5350000e-01, 7.0700000e-01, 1.4140000e+00, 2.8280000e+00, 5.6600000e+00, 1.1300000e+01, 2.2600000e+01],
+               [1.4561822e-05, 1.4224456e-04, 8.7870935e-04, 2.4575511e-03, 8.0615599e-03, 2.3743599e-02, 4.8707533e-02, 7.0969056e-02, 7.6622487e-02, 2.6998756e-02, 4.6235290e-03]],
+    (7,10):[[4.4187500e-02, 8.8375000e-02, 1.7670000e-01, 3.5350000e-01, 7.0700000e-01, 1.4140000e+00, 2.8280000e+00, 5.6600000e+00, 1.1300000e+01, 2.2600000e+01, 4.5200000e+01, 7.0700000e+01],
+            [9.1004029e-03, 4.7514014e-02, 1.5787725e-01, 3.1426661e-01, 6.3241827e-01, 1.0667690e+00, 1.2675411e+00, 1.1326387e+00, 8.3962478e-01, 4.1229082e-01, 1.6311586e-01, 5.6992659e-02]],
+    (7,100):[[2.2097000e-02, 4.4187500e-02, 8.8375000e-02, 1.7670000e-01, 3.5350000e-01, 7.0700000e-01, 1.4140000e+00, 2.8280000e+00, 5.6600000e+00, 1.1300000e+01, 2.2600000e+01],
+             [5.8736062e-04, 3.2124167e-03, 1.5314560e-02, 3.3624616e-02, 6.4082408e-02, 1.3541913e-01, 1.8690009e-01, 1.8862548e-01, 1.1841223e-01, 4.8641263e-02, 1.3723779e-02]],
+    (6,3000):[[5.0000000e-03, 4.5000000e-02, 3.0000000e-01, 1.3000000e+00, 6.0000000e+00],
+              [9.8100000e-10, 7.8480000e-07, 4.9050000e-06, 4.9050000e-06, 7.8480000e-07]],
+    (7,3000):[[1.5000000e-03, 2.1000000e-02, 4.5000000e-02, 9.0000000e-02, 8.0000000e-01, 6.0000000e+00],
+              [1.4715000e-09, 7.8480000e-06, 2.9430000e-05, 3.9240000e-05, 1.4715000e-05, 9.8100000e-07]],
+    (8,3000):[[1.5000000e-03, 1.0000000e-02, 4.5000000e-02, 9.0000000e-02, 3.0000000e+00, 6.0000000e+00],
+              [9.8100000e-09, 3.9240000e-06, 1.1772000e-04, 1.9620000e-04, 4.9050000e-06, 1.9620000e-06]],
+    (8,3000,'Bolivia'):[[4.5000000e-02, 1.0000000e+00, 6.0000000e+00],
```

Comment:
dict keys should be consistent.. get rid of `'Bolivia'`?

Reply:
OK but I regret to alter the original data set.